MacroRecorderGUI/MainWindow.xaml.cs

```diff
@@ -165,5 +173,41 @@ private void AbortPlayback_Click(object sender, RoutedEventArgs e)
         {
             InjectAndCaptureDll.InjectEventAbort();
         }
+
+        private void InitFakeEventsColl() {
+            //todo: use constants. @alexey - should we redefine c++ VKs? why can't we access it with an alternative c# api?
```

Comment:
There's a [KeyInterop](https://docs.microsoft.com/en-us/dotnet/api/system.windows.input.keyinterop?view=netframework-4.7.2) class, which converts between WPF key constants and VK constants.
Is it a good fit here?
Also, I agree, you can have a list of WPF keys you like, and convert them to a list of vKeys:
```suggestion
Key[] keys = {Key.LeftCtrl, Key.LeftShift};
var vKeys = keys.Select(KeyInterop.VirtualKeyFromKey);
```